providers/amazon/tests/unit/amazon/aws/log/test_cloudwatch_task_handler.py

```diff
@@ -47,6 +48,8 @@
 from tests_common.test_utils.config import conf_vars
 from tests_common.test_utils.version_compat import AIRFLOW_V_3_0_PLUS
 
+settings.LOG_FORMAT = "%(message)s"
```

Comment:
This is going to leak state to other tests so isn't right.

Additionally why is this needed? What does it do?